elodie/geolocation.py

```diff
@@ -146,8 +146,6 @@ def place_name(lat, lon):
                 #  set the most specific as the default.
                 if('default' not in lookup_place_name):
                     lookup_place_name['default'] = address[loc]
-
-    if(lookup_place_name is not {}):
```

Comment:
Still have some questions about omitting this `if`. Since `lookup_place_name` is set to `{}` and the loop only sets new keys then by the time it gets to this `if` it can only be an empty dict or a populated dict.

In the case that it's empty we don't want to store it. Not storing it is the only thing the `if` was meant to avoid.

Pls correct me if I'm missing something.